src/AltinnCore/Runtime/RestControllers/InstancesController.cs

```diff
@@ -0,0 +1,241 @@
+    /// </summary>
+    [Route("{org}/{app}/instances")]
+    [Authorize]
+    [ApiController]
+    public class InstancesController : ControllerBase
+    {
+        private readonly ILogger<InstancesController> logger;
+        private readonly HttpClient storageClient;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="InstancesController"/> class
+        /// </summary>
+        public InstancesController(
+            ILogger<InstancesController> logger,
+            IHttpClientAccessor httpClientAccessor)
+        {
+            this.logger = logger;
+            this.storageClient = httpClientAccessor.StorageClient;
+        }
+
+        /// <summary>
+        ///  Gets one application instance from platform storage.
+        /// </summary>
+        /// <param name="instanceOwnerId">the instance owner id (partyId)</param>
+        /// <param name="instanceGuid">the instance guid</param>
+        /// <returns>the instance</returns>
+        [HttpGet("{instanceOwnerId:int}/{instanceGuid:guid}")]
+        [Produces("application/json")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]        
+        public async Task<ActionResult> Get(int instanceOwnerId, Guid instanceGuid)
+        {
+            string instanceId = $"{instanceOwnerId}/{instanceGuid}";
+            Uri storageUrl = new Uri($"instances/{instanceId}", UriKind.Relative);
+
+            HttpResponseMessage httpResponse = await storageClient.GetAsync(storageUrl);
+            if (httpResponse.IsSuccessStatusCode)
+            {
+                string jsonContent = await httpResponse.Content.ReadAsStringAsync();
+                Instance instance = JsonConvert.DeserializeObject<Instance>(jsonContent);
+                GetAndSetAppSelfLink(instance);
+
+                return Ok(instance);
+            }
+
+            return StatusCode((int)httpResponse.StatusCode, httpResponse.ReasonPhrase);
+        }
+
+        /// <summary>
+        ///  Updates an application instance in platform storage.
+        /// </summary>
+        /// <param name="org">the organisation id, the owner of the app</param>
+        /// <param name="app">the app name</param>
+        /// <param name="instanceOwnerId">the instance owner id (partyId)</param>
+        /// <param name="instanceGuid">the instance guid</param>
+        /// <param name="instance">the instance with attributes that should be updated</param>
+        /// <returns>the updated instance</returns>
+        [HttpPut("{instanceOwnerId:int}/{instanceGuid:guid}")]
+        [Produces("application/json")]
+        [Consumes("application/json")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        public async Task<ActionResult> Put(
+            [FromRoute] string org,
```

Comment:
does org and app need to be defined here when they are not used in the function?